reports/host-report.js

```diff
@@ -263,17 +261,11 @@ async function HostReport(year, month, hostId) {
       });
 
       if (!transactions || transactions.length === 0) {
-        throw new Error('No transaction found');
+        console.log(`No transaction found for ${host.slug}, skipping`);
```

Comment:
This is to clear the logs, the previous version was polluted with unnecessary stacktraces

![image](https://user-images.githubusercontent.com/1556356/124571812-6c3ed200-de48-11eb-937e-6425c6790cf3.png)